src/utils/types.ts

```diff
@@ -0,0 +1,3 @@
+import { Client, WalletActions, PublicActions } from 'viem';
+
+export type ViemClient = Client & WalletActions & PublicActions;
```

Comment:
We could be more explicit with the type name using something like: 

`type PublicWalletViemClient` or so but, given that it is the only client that we are exposing, I would keep it shorter.

Reply:
Update: renamed to `PublicWalletClient`